.env.example

```
@@ -0,0 +1,41 @@
+BCSD_DATA_POSTGRES_USER=bcsd
+BCSD_DATA_POSTGRES_PASSWORD=change_me
+BCSD_DATA_AIRFLOW_DB=airflow_metadata
+BCSD_DATA_SUPERSET_DB=superset_metadata
+
+# Airflow
+AIRFLOW_UID=50000
+AIRFLOW__CORE__EXECUTOR=LocalExecutor
+AIRFLOW__CORE__LOAD_EXAMPLES=false
+AIRFLOW__WEBSERVER__EXPOSE_CONFIG=false
+AIRFLOW__WEBSERVER__SECRET_KEY=change_me
+AIRFLOW_ADMIN_USERNAME=admin
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AIRFLOW_ADMIN_USERNAME key should go before the AIRFLOW_UID key

(UnorderedKey)